imageroot/actions/configure-module/validate-input.json

```diff
@@ -243,6 +243,14 @@
                     "pattern": "^$"
                 }
             ]
+        },
+        "deepgram_api_key": {
+            "description": "Deepgram API key for voice recognition",
+            "type": "string"
+        },
+        "openai_api_key": {
+            "description": "OpenAI API key for call summarization",
+            "type": "string"
```

Comment:
The same validation of the `get-configuration` must be enforced here also.

Definition of `satellite_call_transcription_enabled` and `satellite_voicemail_transcription_enabled` are missing